transport/cluster.go

```diff
@@ -36,6 +40,34 @@ type topology struct {
 	racksInDC RacksInDCMap
 	allNodes  []*Node
 	ring      btree.BTree
+	keyspaces KeyspacesMap
+}
+
+type keyspace struct {
+	strategy strategy
+	// TODO: Add and use attributes below
+	// tables		      map[string]table
+	// user_defined_types map[string](string, cqltype)
+}
+
+type stratType int
+
+const (
+	simple          stratType = 0
+	networkTopology           = 1
+	local                     = 2
+	other                     = 3
+)
+
+type strategy struct {
+	strat stratType
```

Comment:
Since linter does not like shorter version, maybe we should use whole strategy?

Reply:
Maybe just 'type'? Looking at current strategy type accessing it would be more readable:
Now accessing if rom host selecttion policy looks like:
```go 
if qi.topology.keyspaces[qi.ksName].strategy.strat == ...
```
Your proposition
```go 
if qi.topology.keyspaces[qi.ksName].strategy.strategy == ...
```
With 'type':
```go 
if qi.topology.keyspaces[qi.ksName].strategy.type == ...
```
Having already such a long access maybe making a function out of it would be nice, what do you think @Michal-Leszczynski ?

Reply:
Maybe let's do it like:
```go
type strategy struct {
	stratType
        ...
```
accessing strategy type from host selection policy looks now like:
```go
if qi.topology.keyspaces[qi.ksName].strategy.strat == simple {
	...
```
after this change would be:
```go
if qi.topology.keyspaces[qi.ksName].strategy.stratType == simple {
	...
```
Having such a long access line how would you think if we add a function to do that @Michal-Leszczynski ?

Reply:
I have applied your suggestion, thanks!